acvm_js/src/execute.rs

```diff
@@ -71,14 +77,14 @@ impl BlackBoxFunctionSolver for SimulatedBackend {
 /// @returns {WitnessMap} The solved witness calculated by executing the circuit on the provided inputs.
 #[wasm_bindgen(js_name = executeCircuit, skip_jsdoc)]
 pub async fn execute_circuit(
+    backend: &SimulatedBackend,
```

Comment:
Can you revert this change and create a new function which takes an explicit black box function solver? This will make this change non-breaking.

Reply:
Created above-mentioned new function, and then changed `execute_circuit` to just initialize a solver and call `execute_circuit_with_black_box_solver()` to avoid nearly full duplication of the function logic.